sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -78,37 +79,19 @@ directive:
 ```yaml
 directive:
   - from: swagger-document
-    where: $.definitions
```

Comment:
Happy to have better support for name overrides, making these transforms way simpler :smile: